Gemfile

```diff
@@ -2,6 +2,6 @@ source "https://rubygems.org"
 
 ruby ">= 3"
 
-gem "fastlane"
+gem "fastlane", "2.201.1"
```

Comment:
The fix has not been merged, so maybe the below will help. 
`gem "fastlane", git: git@github.com:kentrh/fastlane.git, branch: bug/20012-scan-cannot-find-xcresult-testing-swift-package`

Reply:
okay, i've created a branch with the fix
`gem "fastlane", git: git@github.com:KazuCocoa/fastlane.git, branch: scan-swift-package`